config/testgrid/dashboards.yaml

```diff
@@ -9,6 +9,11 @@ dashboard_groups:
   - cert-manager-presubmits-master
   - cert-manager-testing-janitors
 
+dashboard_groups:
```

Comment:
It seems like there is a duplicate key here, is this file auto-generated?

Reply:
Apologies, just an oversight on my end! Fixed 👍